src/app/exercise/exercises/utility/base-exercises/BaseTonalExercise.ts

```diff
@@ -67,30 +66,6 @@ export abstract class BaseTonalExercise<GAnswer extends string = string, GSettin
 
   abstract getQuestionInC(): Exclude<Exercise.NotesQuestion<GAnswer>, 'cadence'>;
 
-  protected override _getSettingsDescriptor(): Exercise.SettingsControlDescriptor<GSettings>[] {
```

Comment:
Consumers are now responsible for adding this manually using the directive. 
Generally I would like to avoid adding settings without directives, as it's easier to order and share this way